tensorflow_probability/python/stats/kendalls_tau_test.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests Kendall's Tau metric."""
+
+import random
+
+import numpy as np
+from scipy import stats
+
+import tensorflow.compat.v2 as tf
+import tensorflow_probability as tfp
+from tensorflow_probability.python.internal import test_util
+
+@test_util.test_all_tf_execution_regimes
+class KendallsTauTest(test_util.TestCase):
```

Comment:
Can we also add a short test or two for `_iterative_mergesort` specifically? (even though it's not exposed publicly, you can import the module `from tensorflow_probability.python.stats import kendalls_tau` and call `kendalls_tau._iterative_mergesort`). It's a relatively complicated method with (fortunately) a clear contract, which makes it worth testing IMHO.

Reply:
Done.